server/datastore/mysql/policies.go

```diff
@@ -1186,9 +1186,9 @@ func (ds *Datastore) GetTeamHostsPolicyMemberships(
 		h.hardware_serial AS host_hardware_serial
 	FROM hosts h
 	LEFT JOIN (
-		SELECT host_id, BIT_AND(COALESCE(passes, 0)) AS passing
+		SELECT host_id, BIT_AND(passes) AS passing
 		FROM policy_membership
-		WHERE policy_id IN (?)
+		WHERE policy_id IN (?) AND passes IS NOT NULL
```

Comment:
Looks good.

Double checking:

Does this work with the following scenario?
1. A host has two policies, one is failing (with valid SQL) and another one has invalid SQL.
2. Calendar event is created.
3. The failing policy with valid SQL is removed from the host.
4. Next calendar cron should delete the event.

Reply:
Yes, works as expected with manual testing.